src/hooks/settingsHooks/useUpdateSettings.ts

```diff
@@ -47,6 +47,14 @@ const convertfromFe = ({
     (_value, key) => FE_TO_BE[key]
   )
 
+  // Remove trailing slash in site URL if present
+  if (
+    "url" in renamedBasicSettings &&
+    (<string>renamedBasicSettings.url).endsWith("/")
+  ) {
+    renamedBasicSettings.url = (<string>renamedBasicSettings.url).slice(0, -1)
+  }
```

Comment:
ts casting uses the `as` keyword; we should also add comments to prove that the type casting is safe as it might not be clear at a glance

```suggestion
  if (
    !!renamedBasicSettings.url &&
    // This is a safe cast as the `url` key gets mapped
    // into a string if it is present and the property
    // is also a string
    (renamedBasicSettings.url as string).endsWith("/")
  ) {
    renamedBasicSettings.url = (renamedBasicSettings.url as string).slice(0, -1)
  }```

Reply:
Fixed! Thanks for pointing this out!